.travis.yml

```diff
@@ -135,7 +151,9 @@ install:
         pip uninstall cython -y;
         pip install dist/scikit-image-*.tar.gz;
       else
-        pip install --no-build-isolation .;
+        # installing in editable mdoe avoids many downloads from pooch
```

Comment:
These Travis changes are confusing to me. Why does this avoid many factors downloads?

Also note typo.

Reply:
Test datasets aren't shipped anymore. 

The pooch logic will search for the data next to the `py` files, so in editable mode, they are all present.